.scripts/Add-Customizations.ps1

```diff
@@ -15,7 +15,7 @@ function Update-Set-Accessors {
             if ($propertyDeclaration -like "*List<*" -or $propertyDeclaration -like "*Dictionary<*") {
                 $newPropertyDeclaration = "$propertyDeclaration{ get; }"
             } else {
-                $newPropertyDeclaration = "$propertyDeclaration{ get; init; }"
+                $newPropertyDeclaration = "$propertyDeclaration{ get; set; }"
```

Comment:
Instead of modifying this part of the script, does it work if you just delete it entirely? If so, you can remove its invocation from the `Invoke-CodeGen.ps1` script too.

Reply:
No, I tried removing the script, but if we remove it, code gen adds setters to collection properties.

![image](https://github.com/user-attachments/assets/9740701f-e339-407e-8db9-347fb2d517bc)

I think we should fix this in the generator, and then we can remove the script.

Reply:
@trrwilson : Do you know if the internal property `SerializedAdditionalRawData` is supposed to be settable?

Reply:
After removing the script, in 90 files, only these 2 properties are updated:

- `internal IDictionary<string, BinaryData> SerializedAdditionalRawData { get; set; }`
- `public IDictionary<string, string> Metadata { get; set; }`

`SerializedAdditionalRawData` is internal, but `Metadata` is a public property."

Reply:
@m-nash / @ArcturusZhang - Are the generated setters for the `SerializedAdditionalRawData` and `Metadata` properties intentional, or is this a bug in code gen?

Reply:
We(OpenAI) asked for `SerializedAdditionalRawData` to be made internal and have a setter, so this is intentional. As for `Metadata`, it seems this is not a generator issue. Nullable collections will have generated setters. This is the only case where a collection property has setters. That is why `Metadata` has a setter.